vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManagerTest.java

```diff
@@ -158,12 +289,250 @@ public void test_updateContentOrder_forNonExistentId() throws BlockDoesNotExistE
         ContentBlock contentBlock1 = new ContentBlock();
         contentBlock1.setId("notARealId");
         contentBlock1.setContentOrder(Integer.valueOf(1));
-        
+
         List<ContentBlock> contentBlocks = new ArrayList<>();
         contentBlocks.add(contentBlock1);
-        
+
         when(contentBlockRepository.findById("notARealId")).thenReturn(Optional.empty());
         managerToTest.updateContentOrder(contentBlocks);
     }
 
+    @Test
+    public void test_getAllContentBlocks_success() throws BlockDoesNotExistException {
+
+        String slideId = "slide1";
+        ISlide slide = new Slide();
+        slide.setId(slideId);
+        ContentBlock firstContentBlock = new ContentBlock();
+        firstContentBlock.setId("contentBlockId1");
+        firstContentBlock.setContentOrder(Integer.valueOf(3));
+
+        ContentBlock secondContentBlock = new ContentBlock();
+        secondContentBlock.setId("contentBlockId2");
+        secondContentBlock.setContentOrder(Integer.valueOf(4));
+
+        List<IContentBlock> contentBlocks = new ArrayList<>();
+        contentBlocks.add(firstContentBlock);
+        contentBlocks.add(secondContentBlock);
+        slide.setContents(contentBlocks);
+        when(slideManager.getSlide(slideId)).thenReturn(slide);
+        when(slideRepo.findById("notARealId")).thenReturn(Optional.empty());
+
+        List<IContentBlock> contentBlock = managerToTest.getAllContentBlocks(slideId);
+        assertTrue(!contentBlock.isEmpty());
+
+    }
+
+    @Test
+    public void test_deleteChoiceBlockById_success() throws BlockDoesNotExistException {
+        String blockId = "realBlockId";
+        String slideId = "realSlideId";
+        when(contentBlockRepository.findById(blockId)).thenReturn(Optional.of(contentBlock));
+        managerToTest.deleteChoiceBlockById(blockId, slideId);
+
+    }
+
+    @Test
+    public void test_deleteChoiceBlockById_nullBlockId() throws BlockDoesNotExistException {
+        String blockId = null;
+        String slideId = "realSlideId";
+        managerToTest.deleteChoiceBlockById(blockId, slideId);
```

Comment:
see above

Reply:
there seem to be more test cases for `deleteChoiceBlockById` further down. They should all be together.